extension/neo4j/src/function/neo4j_migrate.cpp

```diff
@@ -128,8 +129,8 @@ static std::unique_ptr<TableFuncBindData> bindFunc(ClientContext* /*context*/,
     auto password = input->getLiteralVal<std::string>(2);
     auto cli = std::make_shared<httplib::Client>(url, 7474);
     cli->set_basic_auth(userName, password);
-    cli->set_connection_timeout(std::chrono::seconds(1000));
-    cli->set_read_timeout(std::chrono::seconds(1000));
+    cli->set_connection_timeout(std::chrono::seconds(30));
```

Comment:
We should give enough time for neo4j apoc extension to export all the nodes and rels.
For example, it is possible that neo4j takes 2mins to export a node to csv.

Reply:
Okay, how about just `set_connection_timeout` to be 30s then?

I did face this issue once where the CLI got stuck for a long time, as mentioned in https://github.com/kuzudb/kuzu/issues/5341.